runtime/amplitude/src/lib.rs

```diff
@@ -933,6 +1036,127 @@ where
 	}
 }
 
+pub struct CurrencyConvert;
+
+impl currency::CurrencyConversion<currency::Amount<Runtime>, CurrencyId> for CurrencyConvert {
+	fn convert(
+		amount: &currency::Amount<Runtime>,
+		to: CurrencyId,
+	) -> Result<currency::Amount<Runtime>, DispatchError> {
+		Oracle::convert(amount, to)
+	}
+}
+parameter_types! {
+	pub const RelayChainCurrencyId: CurrencyId = XCM(0); // 0 is the index of the relay chain in our XCM mapping
+}
+impl currency::Config for Runtime {
+	type UnsignedFixedPoint = UnsignedFixedPoint;
+	type SignedInner = SignedInner;
+	type SignedFixedPoint = SignedFixedPoint;
+	type Balance = Balance;
+	type GetNativeCurrencyId = NativeCurrencyId;
```

Comment:
This should be removed, it is not part of the latest change to Spacewalk anymore.

Reply:
Good catch, thanks